### DISCUSSION TITLE
How long between OpenStreetMap data exchanges?

### DISCUSSION THREAD

**Discussion by mds08011** [organicmaps/organicmaps#714] [Q&A] [answered]
I added a place to the map in the Organic Maps Android app but have not seen it added to OpenStreetMap yet.  How long does this process typically take?

#### "Bank Of America" added to Organic Maps
<img src="https://user-images.githubusercontent.com/8010709/123116423-c033ca80-d3f5-11eb-9f69-654c94425826.jpeg" width="50%">

#### Area in OpenStreetMap 
(https://www.openstreetmap.org/#map=19/32.82239/-117.10032)
<img src="https://user-images.githubusercontent.com/8010709/123116382-b742f900-d3f5-11eb-9b9d-1b4b5b0e804c.png" width="50%">

<img src="https://user-images.githubusercontent.com/8010709/123116721-025d0c00-d3f6-11eb-937f-1c12bb69c846.png" width="50%">

**Comment by poperigby** [comment]
I'm also interested in this. I added a new business and edited another, but it's not showing up on [openstreetmap.org](openstreetmap.org).

**Comment by biodranik** [comment, marked as answer]
The sync should be quick, usually when the app goes into the background. But there was a bug, which was already fixed and published into Google Play and FDroid. Please wait for the update.

**Reply by mds08011** [reply]
Thank you! I am still on the 2021-06-20 version.

![signal-2021-06-25-132128.jpg](https://user-images.githubusercontent.com/8010709/123481421-8d85ff80-d5b8-11eb-8752-009e2fac78b3.jpg)

**Reply by mds08011** [reply]
I received the 2021.06.23 update via Google Play Store and my edits were added to OpenStreetMap.  Thank you!

**Comment by poperigby** [comment]
I have the lastest version, and I'm logged into my OSM account. I've let the app sit in the background, but my OSM history isn't changing.

**Reply by biodranik** [reply]
The correct version is 2021.06.23
You may try to join a beta program in Google Play and get it.

**Reply by poperigby** [reply]
Hmm. I'm not seeing an option to join a beta program.

**Reply by biodranik** [reply]
Beta testing release is also in review :(

**Reply by poperigby** [reply]
Ahh. Well, I'll jump on that when it's available.
